pool/connection_pool_test.go

```diff
@@ -2215,6 +2215,35 @@ func TestDo(t *testing.T) {
 	require.NotNilf(t, resp, "response is nil after Ping")
 }
```

Comment:
It seems that expected approach is to write commit message as `subsystem: message` and `bugfix` is not a subsystem. It should something like `pool: fix race condition at GetNextConnection()`

Reply:
Done.